ChangeLog.d/add-aes-cbc-to-pkcs5-pbes2.txt

```diff
@@ -0,0 +1,3 @@
+Features
+   * Add support for using AES-CBC 128, 192, and 256 bit schemes
+     with PKCS#5 PBES2.
```

Comment:
Perhaps mention that this means keys encrypted this way can now be parsed by PK parse? That might not be obvious to all users.